backend/src/packages/groups/group.repository.ts

```diff
@@ -0,0 +1,84 @@
+import { and, eq } from 'drizzle-orm';
+
+import { type IRepository } from '~/libs/interfaces/interfaces.js';
+import { type IDatabase } from '~/libs/packages/database/database.js';
+import { type DatabaseSchema } from '~/libs/packages/database/schema/schema.js';
```

Comment:
let's export `type DatabaseSchema` from `/src/libs/packages/database/database.ts` and import it as 
`import { type DatabaseSchema, type IDatabase } from '~/libs/packages/database/database.js';`